### ISSUE TITLE
5.4.0 and 5.5.0 crash during rendering

### ISSUE THREAD

**Issue by govorov** [wojtekmaj/react-datetime-picker#392]
Hi, I'm getting this error when trying to use the last two versions:

```
Error: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: object. Check the render method of `DateTimeInput`.
```

5.3.0 seems to be fine.

**Comment by adi-uchiha** [comment]
Same here

**Comment by adi-uchiha** [comment]
Can you tell me how to use older version because I need to implement this package urgently

**Comment by Jonathanthedeveloper** [comment]
Same here

**Comment by wojtekmaj** [comment]
> Hi, I'm getting this error when trying to use the last two versions:
> 
> ```
> Error: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: object. Check the render method of `DateTimeInput`.
> ```
> 
> 5.3.0 seems to be fine.

What's your setup? I need to know this to help out. Preferably reproducible example. Looks like your setup doesn't read the package well.

Do:

```
console.log(DateTimePicker);
```

and I'm pretty sure what will get logged is an object with the `default` key.

Workaround would be to do something like 

```
import dateTimePickerModule from 'react-datetime-picker';

const { default: DateTimePicker } = dateTimePickerModule;
```

and use that, but remember it's NOT a recommended way of going around the issue and might just get you going in the meantime while we figure things out.

**Comment by wojtekmaj** [comment]
> Can you tell me how to use older version because I need to implement this package urgently
> 

https://docs.npmjs.com/about-semantic-versioning#using-semantic-versioning-to-specify-update-types-your-package-can-accept

**Comment by jotatoledo** [comment]
Broken when using it with nextjs@13.4.12

**Comment by wojtekmaj** [comment]
Fixed in 5.5.1.
